projects/kit/internal/primitive-calendar-range/primitive-calendar-range.component.ts

```diff
@@ -148,7 +148,7 @@ export class TuiPrimitiveCalendarRangeComponent implements OnInit {
         }
     }
 
-    private updatedViewedMonthSecond(month: TuiMonth) {
+    private updatedViewedMonthSecond(month: TuiMonth): TuiMonth | TuiDay {
```

Comment:
i am not sure but I think `updatedViewedMonthSecond` and `updatedViewedMonthFirst` should return just `TuiMonth`